types/provinces/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-var-requires": false
```

Comment:
Why do you need this rule?

Reply:
In attempt to make the test as close to the documentation I wanted to use `const provinces = require('provinces');` (see [docs](https://github.com/substack/provinces#methods)) However after adding that rule I was faced with `Cannot find name 'require'.` so I used `import * as provinces from 'provinces';`

Should `Cannot find name 'require'.` be solved or should the `no-var-requires` be removed?